app/helpers/wally_integration.py

```diff
@@ -0,0 +1,57 @@
+import os
+import requests
+
+from app.models import Review, User, DiveShop, Spot
+
+wally_api_base = os.environ.get('WALLY_API')
+wally_auth_token = os.environ.get('WALLY_AUTH_TOKEN')
+
+def mint_nft(current_review: Review, dive_shop: DiveShop, beach: Spot, user: User):
+  payload = {
+    'walletId': f'user_{str(user.id)}',
+    'metadata': {
+    'image': dive_shop.stamp_uri,
+    'description': 'dive shop name signed via dive shop',
```

Comment:
I forgot to clarify this part - we need to generate a signature from the dive shop wallet using the "Sign a message" endpoint, and then add that as the description. not just this text explicilty

https://docs.wallylabs.xyz/#operation/Sign%20a%20message